insolar/logger.go

```diff
@@ -59,4 +120,7 @@ type Logger interface {
 	WithFields(map[string]interface{}) Logger
 	// WithField return copy of Logger with predefined single field.
 	WithField(string, interface{}) Logger
+
+	// Copy copies logger
+	Copy() Logger
```

Comment:
This is wrong. You don't need copy if thing is immutable and all methods do CopyOnWrite

Reply:
Removed Copy, Set -> With